opus/application/static_media/js/search.js

```diff
@@ -134,9 +134,17 @@ var o_search = {
             let slug = $(this).attr("name");
             let currentValue = $(this).val().trim();
             let values = [];
-            values.push(currentValue);
-            opus.selections[slug] = values;
+            // values.push(currentValue);
+            // opus.selections[slug] = values;
 
+            if (currentValue) {
```

Comment:
I'm not sure I understand why you need to create an array w/a single var to set opus.selections[slug] - wouldn't it be simpler to do something like:
if (currentValue) {  // currentValue.length > 0 ..?
   opus.selections[slug] = [currentValue];
} else { ...

Reply:
Thanks for the prompt feedback.
- After modifying the update action of opus.selections in search.js, opus.selections and modifiedSelections will be exactly the same, so we can use o_utils.areObjectsEqual. checkIfSelectionsObjectsAreTheSame function is removed.
- Regarding to the logic of updating opus.selections, it's been updated based on feedback.